README.md

```diff
@@ -32,5 +32,38 @@ generated_java_libraries()
 Then you can access any of the Java library targets in `generate_workspace.bzl`,
 which export each jar's dependencies as well as the jar itself.
 
+## `transitive_maven_jar` 
+
+After the initial migration, one can use the `transitive_maven_jar` repository rule
```

Comment:
Faulty thinking on my part: I was thinking that if you updated a dep, it would have to redownload everything.

However, thinking about this further, `transitive_maven_jar(name = "foo")` should be able to copy generate_workspace.bzl file in /ob/external/foo, so people should be able to do:

```
transitive_maven_jar(
    name = "foo",
    ...
)

load('@foo//:generate_workspace.bzl', 'generated_maven_jars')
generated_maven_jars()
```

Please add that to the code example.